Subject: remove AVAILABLE_VETH= from /proc/iSeries/config

All vio devices on legacy iseries are now available via sysfs.
According to google, nothing beside hwinfo uses AVAILABLE_VETH=
So it is safe to remove the line from /proc/iSeries/config and
let everyone use sysfs to discover the configured virtual network interfaces.


Signed-off-by: Olaf Hering <olh@suse.de>

---
 arch/powerpc/platforms/iseries/viopath.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/arch/powerpc/platforms/iseries/viopath.c	2007-10-31 17:43:31.000000000 -0400
+++ b/arch/powerpc/platforms/iseries/viopath.c	2007-10-31 17:43:59.000000000 -0400
@@ -113,7 +113,6 @@ static vio_event_handler_t *vio_handler[
 static int proc_viopath_show(struct seq_file *m, void *v)
 {
 	char *buf;
-	u16 vlanMap;
 	dma_addr_t handle;
 	HvLpEvent_Rc hvrc;
 	DECLARE_COMPLETION(done);
@@ -141,7 +140,6 @@ static int proc_viopath_show(struct seq_
 
 	wait_for_completion(&done);
 
-	vlanMap = HvLpConfig_getVirtualLanIndexMap();
 
 	buf[HW_PAGE_SIZE-1] = '\0';
 	seq_printf(m, "%s", buf);
@@ -150,7 +148,6 @@ static int proc_viopath_show(struct seq_
 			 DMA_FROM_DEVICE);
 	kfree(buf);
 
-	seq_printf(m, "AVAILABLE_VETH=%x\n", vlanMap);
 
 	node = of_find_node_by_path("/");
 	sysid = NULL;
